### ISSUE TITLE
Binary operator '|=' cannot be applied to two 'UIAccessibilityTraits' operands

### ISSUE THREAD

**Issue by ghost** [louisdh/huekit#6]
![screen shot 2018-12-19 at 4 07 18 pm](https://user-images.githubusercontent.com/39147630/50215161-316b1080-03a8-11e9-9ce8-5bc9e0664f97.png)
Xcode 10.1 , Swift 4.2

**Comment by ghost** [comment]
Solution 

```swift

#if swift(>=4.2)
    public extension UIAccessibilityTraits {
        public static func |= (left: inout UIAccessibilityTraits, right: inout UIAccessibilityTraits) {
            left = left | right
        }

        public static func | (left: UIAccessibilityTraits, right: UIAccessibilityTraits) -> UIAccessibilityTraits {
            return UIAccessibilityTraits(rawValue: left.rawValue | right.rawValue)
        }

        public static prefix func ~ (trait: UIAccessibilityTraits) -> UIAccessibilityTraits {
            return UIAccessibilityTraits(rawValue: ~trait.rawValue)
        }
    }
#endif

```
